src/__tests__/App.test.tsx

```diff
@@ -3,18 +3,24 @@ import { act } from 'react-dom/test-utils';
 
 import App from '../components/App';
 import { render } from './utils/setupTests';
-import { screen } from './utils/test-utils';
+import { screen, FetchMockSandbox } from './utils/test-utils';
 
 describe('App', () => {
+  beforeEach(() => {
+    (global.fetch as FetchMockSandbox).get(
+      'glob:https://treeherder.mozilla.org/api/project/*/push/*',
+      { results: [] },
+    );
+  });
+
   test('Should render search view on default route', async () => {
     render(<App />);
 
     // Title appears
     expect(screen.getByText(/PerfCompare/i)).toBeInTheDocument();
 
-    await act(async () => void jest.runOnlyPendingTimers());
+    act(() => void jest.runAllTimers());
```

Comment:
You'll see similar changes in later commits too, so let me explain here:

1. I generally find `runAllTimers` easier to use and less error prone than `runOnlyPendingTimers`, because it will run all timers, even the ones that are set as a result of running a setTimeout callback. We might enter into infinite loops in some cases because of that, and that's only when `runOnlyPendingTimers` is useful IMO.
2. The fake timers are synchronous, not asynchronous. So we don't need to use the asynchronous version of `act`, which is easier to read.